be/src/olap/tablet.h

```diff
@@ -448,6 +450,7 @@ class Tablet : public BaseTablet {
     std::mutex _ingest_lock;
     std::mutex _base_compaction_lock;
     std::mutex _cumulative_compaction_lock;
+    std::mutex _cooldown_compaction_lock;
```

Comment:
We already have `_cold_data_compaction_lock`, what's the different between these two lock?